tests/search.service.spec.js

```diff
@@ -100,8 +100,8 @@ describe("Searching for hackers", function () {
                 })
                 .end(function (err, res) {
                     res.should.have.status(200);
-                    res.body.should.have.property('data');
-                    res.body.data.should.have.length(2);
+                    res.body.should.have.property("data");
+                    res.body.data.should.have.length(6);
```

Comment:
Why did this change?

Reply:
I added some more hackers to test a full team.